cli/route.go

```diff
@@ -118,6 +142,161 @@ func runRoute(args *docopt.Args, client controller.Client) error {
 	return nil
 }
 
+func runRouteConfigGenerate(args *docopt.Args, client controller.Client) error {
+	// list routes for the given apps
+	apps := args.All["<apps>"].([]string)
+	req := api.ListAppRoutesRequest{Apps: apps}
+	var res api.ListAppRoutesResponse
+	if err := client.Invoke("flynn.api.v1.Router/ListAppRoutes", &req, &res); err != nil {
```

Comment:
I assume this will be changed to use the gRPC protocol via the generated client vs gRPC-web once that is exposed?

Reply:
Yes, although adding gRPC support to the router will come in another PR, for now we'll use gRPC web.